src/data/FetchHttpClientRepository.ts

```diff
@@ -13,15 +13,17 @@ import {
     HttpError,
     HttpRequest,
     HttpClientResponse,
+    Credentials,
 } from "../repositories/HttpClientRepository";
 import { joinPath } from "../utils/connection";
+import { PATToken } from "../api/types";
 
 export class FetchHttpClientRepository implements HttpClientRepository {
     constructor(public options: ConstructorOptions) {}
 
     request<Data>(options: HttpRequest): CancelableResponse<Data> {
         const controller = new AbortController();
-        const { baseUrl = "", auth } = this.options;
+        const { baseUrl = "", auth, personalToken } = this.options;
```

Comment:
This adds support for personalToken to the fetch repo, what about the axios repo?